cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -23,3 +23,17 @@ include::modules/developer-cli-odo-installing-odo-on-linux-on-ibm-z.adoc[levelof
 include::modules/developer-cli-odo-installing-odo-on-windows.adoc[leveloffset=+1]
 
 include::modules/developer-cli-odo-installing-odo-on-macos.adoc[leveloffset=+1]
+
+= Installing odo in VS Code
```

Comment:
As Yana mentioned the instructions in this section should be in a separate procedure module.
Once you create the procedure module you add it in this file as an include, for example see line 23 and 24 above. 
This is an Assembly file, where we add only the includes for the files/sections we want under this assembly. Kind of like a chapter with multiple sections.